apps/nova/src/hooks/call.ts

```diff
@@ -357,3 +357,47 @@ export function useCallEvents(
     mateVideo,
   };
 }
+
+export function useFullScreen() {
+  const [isFullScreen, setIsFullScreen] = useState(false);
+
+  const startFullScreen = useCallback(async () => {
+    try {
+      setIsFullScreen(true);
+      await document.body.requestFullscreen();
+    } catch (error) {
+      console.log(error);
```

Comment:
Please use `toaster.error` to show error notification in case the request failed.